gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/DateAdd.feature

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+@StepClassMap @StepDateAdd
+Feature: Step - dateAdd()
+
+  @GraphComputerVerificationInjectionNotSupported
+  Scenario: g_injectXdatetimeXstrXX_dateAddXDT_hour_2X
+    Given the empty graph
+    And the traversal of
+      """
+      g.inject(datetime('2023-08-02T00:00:00Z')).dateAdd(DT.hour, 2)
+      """
+    When iterated to list
+    Then the result should be unordered
+      | result |
+      | dt[1690941600] |
```

Comment:
would it be possible to make the value asserted more human readable? i can't easily tell from reading this if the result is right. it would be nicer if it was more like: `| dt[2023-08-02T02:00:00Z] |`

Reply:
done in https://github.com/apache/tinkerpop/pull/2223/commits/3cc1334abb741e3e193e985182dffc325dda233c